pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/AdminResource.java

```diff
@@ -293,6 +293,9 @@ protected Policies getNamespacePolicies(NamespaceName namespaceName) {
             BundlesData bundleData = pulsar().getNamespaceService().getNamespaceBundleFactory()
                     .getBundles(namespaceName).getBundlesData();
             policies.bundles = bundleData != null ? bundleData : policies.bundles;
+            if (policies.is_allow_auto_update_schema == null) {
```

Comment:
Please add a comment here to indicate to return the broker value here is for keeping compatibility.

Reply:
Done